### DISCUSSION TITLE
Proxmox Docker VE unknown host when trying login via SSH

### DISCUSSION THREAD

**Discussion by JL-EU** [tteck/Proxmox#3374] [Q&A]
Hello
I installed Docker on a Proxmox VE using the Helper-Scripts without issues. An IP address is assigned and I can access the VE when using the IP address. However, I can't access the VE using the hostname of the VE, then I get the error message "unknown host". I have installed VMs on the same Proxmox server and here I never had issues accessing using the hostname. In this example I can ping the IP address 192.168.1.159 while hostname frigate-docker.local gives the message "Unknown host".  The settings are the following:
<img width="334" alt="image" src="https://github.com/user-attachments/assets/34083b32-f775-415d-9c20-d4d619312876">
Any help is appreciated, thanks,

**Comment by Mellowlynx** [comment]
Make sure the host in the host file or in your network DNS to work property.
Else is unknown and can't look up its IP
